Lib-Open5e/properties.xml

```diff
@@ -1,10 +1,10 @@
 <map>
   <entry>
     <string>version</string>
-    <string>1.6.1</string>
+    <string>1.8.3</string>
```

Comment:
I know RPTools has made a fuss about err'ybody getting up to date w/ 1.8.3, but our libraries aren't explicitly requiring 1.8.3 yet, so avoid letting this change in.